x-pack/plugins/alerting/public/lib/test_utils.tsx

```diff
@@ -82,13 +79,11 @@ export const createAppMockRenderer = ({
     },
   };
   const AppWrapper: React.FC<{ children: React.ReactElement }> = React.memo(({ children }) => (
-    <I18nProvider>
```

Comment:
The i18n context provider is not explicitly needed since KibnaRenderContextProvider provides it